source/code-snippets/collation/aggregation-collation.js

```diff
@@ -1,3 +1,4 @@
+// Retrieve the total matching names, grouped by the first_name field and sorted by using the German phonebook collation
```

Comment:
S: since this comment is causing horizontal scroll, make it a multi line comment. Or, could it be more beneficial to comment before the each stage + the collation line?